source/math/calculus_parser.py

```diff
@@ -97,7 +97,7 @@ def _find_pattern(self, query: str, pattern_dict: dict, try_predict: bool) -> bo
 
                     # Check if listed variables are correct
                     for var in symbols:
-                        if not str(var).isalpha():
+                        if not str(var).isalpha() or not str(var).isascii():
                             raise ParseError(f"Variables can only contain letters\nIncorrect variable: '{var}'")
```

Comment:
We allow only latin variables? Then we should tell user about it
```suggestion
                        if not str(var).isalpha() or not str(var).isascii():
                            raise ParseError(f"Variables can only contain latin letters\nIncorrect variable: '{var}'")
```